platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfHomepageGadgetSuggestionGadgetTestIT.java

```diff
@@ -430,7 +426,7 @@ public void test07_checkHiddenOfSuggestionGadgetWhenUserAcceptSuggestionOfUser()
     ELEMENT_GADGET_USER_SUGGESTION.find(byText(DATA_NAME_USER1)).parent().parent().find(ELEMENT_BUTTON_CONNECT_USER_FROM_GADGET)
                                   .waitUntil(Condition.appears, Configuration.timeout)
                                   .click();
-    ELEMENT_GADGET_USER_SUGGESTION.find(byText(DATA_NAME_USER1)).shouldNot(Condition.exist);
+    ELEMENT_GADGET_USER_SUGGESTION.find(byText(DATA_NAME_USER1)).shouldNot(Condition.visible);
```

Comment:
Are the changes of the class related to signout issue ? It seems several things are mixed in this PR.

Reply:
these  changes are removed and it will be done in other branch

Reply:
moved to this PR https://github.com/exoplatform/platform-qa-ui/pull/90